src/mainwindow.h

```diff
@@ -116,7 +116,7 @@ private slots:
     void systemSelectionChangedSignal(); ///< indicate that someone clicked on another system -> we need to reload TreeView and the info box
 
 private:
-    void _addDataToPlot(TreeItem * const item);
+    void _addDataToPlot(TreeItem * item);
```

Comment:
Style: Should be `TreeItem * const item` otherwise signature/definition mismatch (some IDEs don't match their tags anymore).